GVRf/Framework/jni/shaders/material/unlit_shader.cpp

```diff
@@ -85,6 +85,11 @@ void UnlitShader::render(const glm::mat4& mvp_matrix,
         throw error;
     }
 
+    if (texture->getId() == 0) {
+        std::string error = "UnlitShader::render : texture with invalid Id";
```

Comment:
Isn't this misleading? The actual thing that happened is that the object was recycled.